app/controllers/tag_controller.rb

```diff
@@ -41,13 +41,18 @@ def show
                         .page(params[:page])
                         .order("node_revisions.timestamp DESC")
     end
-
     @notes = nodes.where('node.nid NOT IN (?)', qids) if @node_type == "note"
     @questions = nodes.where('node.nid IN (?)', qids) if @node_type == "questions"
     @wikis = nodes if @node_type == "wiki"
     @nodes = nodes if @node_type == "maps"
     @title = params[:id]
     set_sidebar :tags, [params[:id]]
+    
+    respond_with(@nodes) do |format|
```

Comment:
Oops, and i mistyped -- this should be `@notes`, not `@nodes`. Sorry! and on lines 53-4.